<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WindowTranslator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="pointLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>&lt;img src=&quot;:/cursor/cross&quot;/&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="GraphicsViewWithoutWheel" name="shotView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>64</y>
      <width>512</width>
      <height>448</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="refreshLabel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>&lt;img src=&quot;:/icon/refresh&quot;/&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="ocrCheckBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OCR</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="translateCheckBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsViewWithoutWheel</class>
   <extends>QGraphicsView</extends>
   <header>graphicsviewwithoutwheel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
